### PR TITLE
Update ownerless-groups-teams.md

### PR DESCRIPTION
**Pull request by msbemba**
Added related article to the document per Author

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10013

File: microsoft-365/admin/create-groups/ownerless-groups-teams.md [hunk 1]
```diff
@@ -67,3 +67,4 @@ Up to two group members per group can accept the invitation to become an owner.
 ## Related topics
 
 [Ownerless group FAQ](/exchange/troubleshoot/groups-and-distribution-lists/ownerless-group-policy)
+[Search the audit log in the compliance porta](/compliance/search-the-audit-log-in-security-and-compliance)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/ownerless-groups-teams.md]:
```suggestion

[Search the audit log in the compliance portal](/compliance/search-the-audit-log-in-security-and-compliance)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**